build/common.ps1

```diff
@@ -157,95 +157,81 @@ Function Install-DotnetCLI {
     param(
         [switch]$Force
     )
-    $vsMajorVersion = Get-VSMajorVersion
-    $MSBuildExe = Get-MSBuildExe $vsMajorVersion
-    $CliBranchListForTesting = & $msbuildExe $NuGetClientRoot\build\config.props /v:m /nologo /t:GetCliBranchForTesting
-    $CliBranchList = $CliBranchListForTesting.Split(';');
+    $env:DOTNET_HOME = $CLIRoot
+    $env:DOTNET_INSTALL_DIR = $NuGetClientRoot
 
+    Trace-Log "The .NET SDK will be installed in : $CLIRoot"
+
+    $VSVersion = Get-VSMajorVersion
+    $MSBuildExe = Get-MSBuildExe $VSVersion
+    $DotNetExe = Join-Path $CLIRoot 'dotnet.exe'
     $DotNetInstall = Join-Path $CLIRoot 'dotnet-install.ps1'
 
-    #If "-force" is specified, or dotnet.exe under cli folder doesn't exist, create cli folder and download dotnet-install.ps1 into cli folder.
+    # If "-force" is specified, or dotnet.exe under cli folder doesn't exist, create cli folder and download dotnet-install.ps1 into cli folder.
     if ($Force -or -not (Test-Path $DotNetExe)) {
-        Trace-Log "Downloading .NET CLI $CliBranchForTesting"
-
+        Trace-Log "Downloading .NET SDK Install Script"
         New-Item -ItemType Directory -Force -Path $CLIRoot | Out-Null
-
         Invoke-WebRequest 'https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1' -OutFile $DotNetInstall
     }
 
-    ForEach ($CliBranch in $CliBranchList) {
-        $CliBranch = $CliBranch.trim()
-        $CliChannelAndVersion = $CliBranch -split "\s+"
-
-        $Channel = $CliChannelAndVersion[0].trim()
-        if ($CliChannelAndVersion.count -eq 1) {
-            $Version = 'latest'
-        }
-        else {
-            $Version = $CliChannelAndVersion[1].trim()
-        }
+    $Arch = if ([Environment]::Is64BitOperatingSystem) { "x64"; } else { "x86"; }
```

Comment:
I think the only place this is used is when calling dotnet-install.ps1, and that script can already figure out the right architecture, so I think we can get rid of this.